packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -125,8 +125,10 @@ export default decorate([
             type,
           }
           if (type === 's3') {
-            const { bucket, directory } = state
+            const { allowUnauthorized, bucket, directory, protocol = 'https' } = state
             urlParams.path = bucket + '/' + directory
+            urlParams.allowUnauthorized = allowUnauthorized
+            urlParams.protocol = protocol
```

Comment:
I trust you this, the code of this page is horrible: `setInsecure(true) → https`?!!! WTF.

Reply:
That's very smart indeed ^^